### PR TITLE
Added Badge Magic to Apps

### PR DESCRIPTION
**Pull request by adityastic**
#519 

Screenshot: 
Logos:
Screenshot 2019-06-21 at 3 29 59 PM

Apps:
Screenshot 2019-06-21 at 2 26 11 PM

### CODE REVIEW

**Comment by adityastic** [comment]
@mariobehling please review

**Review by mariobehling — CHANGES_REQUESTED** [review]
In order to keep the design consistent, please also add the two other apps, keep the size consistent and ensure all points in the issue are taken care. Thank you

**Comment by adityastic** [comment]
@mariobehling can you give me an example of semi-transparent background with app name on top? so i can visualize your idea well

**Comment by adityastic** [comment]
> add a semitransparent background and text with the app name on the bottom of the app screenshots

@mariobehling I am not able to visualize the design you are looking for, can you explain to me with a screenshot of something which is similar to the expected outcome?

**Comment by kushthedude** [comment]
> > add a semitransparent background and text with the app name on the bottom of the app screenshots
> 
> @mariobehling I am not able to visualize the design you are looking for, can you explain to me with a screenshot of something which is similar to the expected outcome?

@adityastic Since you see there are three apps on one row, I suggest you add Open Event Attendee and Open Event Orga app too

**Review by kushthedude — CHANGES_REQUESTED** [review]
@adityastic Since you see there are three apps on one row, I suggest you add Open Event Attendee and Open Event Orga app too

**Review by kushthedude — CHANGES_REQUESTED** [review]
Optimise the jpg file, 327kb is too high it will increase load time by a good amount

**Comment by adityastic** [comment]
@kushthedude yeah I get that part. But check the issue description, it states to add a semi transparent bg with the app name on the bottom. No idea where to use the background semi translucent. There must be an image inside Mario's head, I want him to explain to me about it

**Comment by mariobehling** [comment]
Please add the other apps first and progress to open questions in a follow up PR afterwards.

**Comment by adityastic** [comment]
Ohk

**Comment by adityastic** [comment]
@mariobehling updated

Screenshot 2019-07-19 at 4 20 11 PM
Screenshot 2019-07-19 at 4 20 17 PM

**Review by mariobehling — CHANGES_REQUESTED** [review]
1. Please don't add the logos except for Badge Magic. There is already a logo of eventyay.
2. All screenshots need to be conform in the same size.

**Comment by adityastic** [comment]
@ShridharGoel give me a recent screenshot of orga app

**Comment by ShridharGoel** [comment]
Dashboard - Orga App

**Comment by ShridharGoel** [comment]
Dashboard - Orga App

**Comment by adityastic** [comment]
@mariobehling updated

Screenshot 2019-07-31 at 2 33 42 PM

**Comment by ShridharGoel** [comment]
@adityastic Why not using the Event Dashboard screenshot for Orga App? The one which I provided.

**Comment by adityastic** [comment]
@ShridharGoel the screen ratio was different. If you can provide me a screenshot with a modern ratio like 19:9, I can use it. All of these are currently from a single device, which helps us keep consistency between screenshot

**Comment by adityastic** [comment]
@mariobehling plz update

**Comment by kushthedude** [comment]
> @mariobehling plz update

@adityastic Please provide Deployment Link.

**Comment by adityastic** [comment]
@kushthedude Already did, please check the pr

**Comment by kushthedude** [comment]
> @kushthedude Already did, please check the pr

Can you please point it out?

File: index.html [hunk 1]
```diff
@@ -551,6 +558,29 @@ <h1>Android Apps of FOSSASIA</h1>
 							</div>
 
 						</div>
+					<div class="row">
+							<div class="col-md-4 col-sm-12 sponsor-column"  >
+								<div class=" text-center" style="margin-bottom: 30px">
+									<a href="https://play.google.com/store/apps/details?id=org.fossasia.badgemagic">
```
Comment (adityastic) [inline index.html:564]: @kushthedude and many more

**Comment by kushthedude** [comment]
I am asking deployment link for the pr buddy

**Comment by adityastic** [comment]
@kushthedude you seriously want me to deploy the app just to show that I added 4 images to the website? Like Seriously?

**Comment by kushthedude** [comment]
Responsiveness Check😏

**Review by abishekvashok — APPROVED** [review]
LGTM

**Comment by adityastic** [comment]
@kushthedude not sure what you meant by responsiveness of images. Please Elaborate

**Pull request merged**